go.mod

```diff
@@ -1,32 +1,26 @@
 module github.com/ONSdigital/dp-search-api
 
-go 1.17
+go 1.16
```

Comment:
This should be changed back to go 1.17

fyi - if you haven't done so already it is worth running `brew update go` this should install version 1.17.2 on your local machine ..... then run `go mod tidy` :)